src/pdf-image-source.cc

```diff
@@ -77,7 +86,9 @@ bool PDFImageSource::LoadAndScale(const DisplayOptions &opts, int frame_offset,
         cairo_surface_t *surface = cairo_image_surface_create_for_data(
             (uint8_t *)image->begin(), kCairoFormat, render_width,
             render_height, stride);
+
         cairo_t *cr = cairo_create(surface);
+        cairo_translate(cr, -selection.x1, -selection.y1);
```

Comment:
I believe the `cairo_translate()` needs to come after the calll to `cairo_scale()`

Reply:
(solved)